### ISSUE TITLE
[Important] Dumb entities

### ISSUE THREAD

**Issue by ArcasCZ** [brandon3055/Draconic-Evolution#292]
Hello there!

We have a big problem with DE. The problem appears when someone try to kill that special dragon. Every time someone tries, there is about 140 000 entities, that has to be deleted manually. Is there any way you could limit lifetime of this entities? Or add simply command, that will remove these entities (we have to use /killall all DIM1, but that removes dragon too). Or add autodetect and delete of this entities? It causes masive lagspikes (10 TPS, 9 TPS).

**Comment by brandon3055** [comment]
What entities are you referring to? The dragons projectiles despawn after 40 seconds

**Comment by ArcasCZ** [comment]
Today I found name of the entities - DragonBolt. They appear in lagre amount, so I have to develop plugin to clear all entities with this name every 60 seconds. Did I mentioned we use KCauldron?

**Comment by brandon3055** [comment]
Well that was unexpected... I never even considered that entity as a possible cause and on first inspection i still didnt see the issue.

But on closer inspection it seems this should have been a problem from day one because the base Entity class dose not increment ticksExisted meaning the entity never gets old enough to be deleted. But somehow some way ticksExisted is being being incremented so it worked. I also have no idea why it dose not work on your server but i suspect its directly related to Cauldron. So i have completely side stepped the issue but adding my own tick counter.

Long story short it should be fixed in the next version.

**Comment by ArcasCZ** [comment]
Every other entity, that has to be despawned is despawned - shooted arrow, dropped item, another projectiles from TConstruct etc. But only DragonBolt is not despawned...

**Comment by ArcasCZ** [comment]
Anyway - is there any available dev build? 'Cause when I try to compile it myslef, I got stucked on few GUI problems...

**Comment by brandon3055** [comment]
i released 1.0.2b last night.

**Comment by ArcasCZ** [comment]
Thanks alot!
